docs/webportal/README.md

```diff
@@ -115,6 +115,15 @@ Click the tab "Cluster View" to see the status of the whole cluster. Specificall
 * Hardware: Hardware metrics of each machine.
 * K8s Dashboard: The Kubernetes Dashboard.
 
+### Virtual cluster management
+
+OpenPAI Virtual Cluster is designed to run jobs as a shared, multi-tenant cluster in an operator-friendly manner while maximizing the throughput and the utilization of the cluster. Click the tab "Virtual Cluster" to  see virtual cluster's status and change virtual clusters for user and admin respectively. Specifically:
+
+* Add a new virtual cluster
+* Remove an obsolete virtual cluster
+* Increase or decrease virtual cluster's capacity
+* Change virtual cluster's availability
```

Comment:
can you explain a bit more in the document about the meaning of vc availability?

Reply:
I also add some description about capacity.